requirements.txt

```diff
@@ -1,2 +1,3 @@
 mako
 PyYAML
+nose
```

Comment:
Both approach are doable, I'd say pick the one you like best :)

The impact here is limited since the list of deps is short, for larger project the impact would be bigger.